### PR TITLE
Update reference to mesos instead of airbnb

### PR DESCRIPTION
**Pull request by elingg**

### CODE REVIEW

**Comment by elingg** [comment]
@lingmann , @brndnmtthws please take a quick look thx!

File: bin/chronos-framework [hunk 1]
```diff
@@ -69,7 +69,7 @@ function run_jar {
                   -Djava.library.path=/usr/local/lib:/usr/lib64:/usr/lib
                   -Djava.util.logging.SimpleFormatter.format="$log_format" )
   # TODO: Set main class in pom.xml and use -jar
-  exec java "${vm_opts[@]}" -cp "$chronos_jar" com.airbnb.scheduler.Main "$@"
+  exec java "${vm_opts[@]}" -cp "$chronos_jar" com.apache.mesos.scheduler.Main "$@"
```
Comment (lingmann) [inline bin/chronos-framework]: Just ran some tests and it looks like this should be: `org.apache.mesos.chronos.scheduler.Main`

Comment (brndnmtthws) [inline bin/chronos-framework]: Yes, that's correct.

Comment (elingg) [inline bin/chronos-framework]: package org.apache.mesos.chronos.scheduler yup. Let me update that typo.

**Comment by elingg** [comment]
should be fixed and working @lingmann. Thanks for your help testing

File: bin/chronos-framework [hunk 2]
```diff
@@ -69,7 +69,7 @@ function run_jar {
                   -Djava.library.path=/usr/local/lib:/usr/lib64:/usr/lib
                   -Djava.util.logging.SimpleFormatter.format="$log_format" )
   # TODO: Set main class in pom.xml and use -jar
-  exec java "${vm_opts[@]}" -cp "$chronos_jar" com.airbnb.scheduler.Main "$@"
+  exec java "${vm_opts[@]}" -cp "$chronos_jar" com.apache.mesos.chronos.scheduler.Main "$@"
```
Comment (brndnmtthws) [inline bin/chronos-framework]: `org` rather than `com` here.

**Comment by elingg** [comment]
too much multitasking today :/ fixed the second typo. Should be good now!

**Comment by brndnmtthws** [comment]
:+1:

**Pull request merged**